tests/tests.go

```diff
@@ -209,6 +209,22 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() time.Duration {
+	managerDeployment, err := testClient.KubeClient.AppsV1().Deployments(managerNamespace).Get(names.MANAGER_DEPLOYMENT, metav1.GetOptions{})
+	Expect(err).ToNot(HaveOccurred(), "Should succeed getting Deployment object")
+	Expect(len(managerDeployment.Spec.Template.Spec.Containers)).To(Equal(1))
+
+	for _, arg := range managerDeployment.Spec.Template.Spec.Containers[0].Args {
+		waitTime := 0
+		if elementsParsed, err := fmt.Sscanf(arg, "--wait-time=%d", &waitTime); elementsParsed == 1 {
```

Comment:
Please split this line to two lines one for initialization and one for checking so it will be easier to review :)

Reply:
I think it's the standard golang way to `if`... at least that's what I saw from other reviews that commented against separating the line to 2 lines. @phoracek - is there a format we should stick to?

Reply:
Most of  `if` clauses in kubemacpool code base wont initialize in the same line
it is possible but I think its confusing.